rules/extra_space_character.yml

```diff
@@ -0,0 +1,10 @@
+id: extra_space_character
+message: Comment or string contains an extra space.
+severity: error
+language: Rust
+# The rule only check for double space to not report errors when whitespace is used for formatting.
```

Comment:
Very minor but maybe `The rule only checks for double space` instead.

Reply:
Fixed.